### ISSUE TITLE
Rework reflow on the mobile devices

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/calendar#4283]
### Steps to reproduce

This is an accessibility issue. 

When using the app on a smaller, vertical screen and activating “New event” a fly-out menu opens from the right. This should open on top of interface, but it opens under the menu field, and the user won’t be able to reach it.

This does not technically fail this WCAG-criterion, though it would be a serious issue for someone trying to use the app on a mobile phone.

### Expected behavior

Popover comes in front (on the top) of the view. 

### Actual behaviour

At the moment it looks like this

![image](https://user-images.githubusercontent.com/6078378/173398247-4fb3ae5b-1de1-467a-899f-ab4b6209818c.png)

### Calendar app version

3.3.2

**Comment by szaimen** [comment]
Does https://github.com/nextcloud/nextcloud-vue/pull/2747 address this?

**Comment by JuliaKirschenheuter** [comment]
> > I've tested it with modal + (not completely) hidden sidebar - and this doesn't work
> 
> Can you post a screenshot of this?

This one https://github.com/nextcloud/calendar/issues/4283#issue-1269669611

**Comment by szaimen** [comment]
> This one [#4283 (comment)](https://github.com/nextcloud/calendar/issues/4283#issue-1269669611)

Okay but this should be fixed with https://github.com/nextcloud/nextcloud-vue/pull/2747

**Comment by JuliaKirschenheuter** [comment]
> Okay but this should be fixed with [nextcloud/nextcloud-vue#2747](https://github.com/nextcloud/nextcloud-vue/pull/2747) due to the z-index change...

Hm, i've tested it, but haven't seen a difference. Could you test it too?

**Comment by szaimen** [comment]
If you have tested it I guess it does not work as expected... Can you show a screenshot of your test? Does it still look the same like your initial report here?

**Comment by JuliaKirschenheuter** [comment]
> Does it still look the same like your initial report here?

yes =(

**Comment by szaimen** [comment]
> yes =(

Then your test does not seem to work... Did you clear all caches and linked the vue repo correctly?

**Comment by szaimen** [comment]
> > yes =(
> 
> Then your test does not seem to work... Did you clear all caches and linked the vue repo correctly?

I am asking because at least the right sidebar should now span the whole width on mobile (below 512px)

**Comment by szaimen** [comment]
This can now be tested again with https://github.com/nextcloud/nextcloud-vue/releases/tag/v6.0.0-alpha.0

**Comment by szaimen** [comment]
Also the new calendar beta release contains this fix now.
